disk/test/com/treode/disk/DiskTestTools.scala

```diff
@@ -40,6 +41,17 @@ private object DiskTestTools {
 
   val sysid = SystemId (0, 0)
 
+  def assertEqNotification (msgs: List[String], notif: Notification) {
```

Comment:
If you define this as `def assertError (msgs: String*) (actual: Notification)`, you save the need for the `List(...)` part at the call sites.